bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -211,3 +209,46 @@ module.exports = GradleBuilder;
 function isAutoGenerated(file) {
     return fs.existsSync(file) && fs.readFileSync(file, 'utf8').indexOf(MARKER) > 0;
 }
+
+/**
+ * A special superspawn-like implementation, required to workaround the issue
```

Comment:
This is not really a consideration anymore now that the redirection has to be more specialized to this particular case